rollup/rollup-cdn.config.js

```diff
@@ -1,76 +1,14 @@
-  importMetaUrlAssetsPlugin,
-  manualChunkOptions,
-  minifyStylesheet,
-  terserPlugin,
-  treeshakeOptions,
-  visualizerPlugin,
-} from './rollup-common.js';
-import { depsManifestPlugin } from './rollup-plugin-deps-manifest.js';
-import { stylesAssetsPlugin } from './rollup-plugin-styles-assets.js';
+import { toCdnEntryName } from '../tasks/cdn-entry-name.js';
+import { CDN_ENVIRONMENTS } from '../tasks/cdn-environments.js';
+import { getCdnRollupConfig } from './rollup-cdn-common.js';
 
-const packageVersion = getVersion();
+export default getCdnRollupConfig(getCdnEntryName(), true);
 
-const sourceDir = 'src';
-const outputDir = 'dist-cdn';
-
-const inputFilesPairs = getMainFiles(sourceDir).map((file) => {
-  const entryPath = path.parse(file).name;
-  return [entryPath, file];
-});
-
-export default {
-  // defines the files to be included into the build: all components + smart manager + translations + i18n
-  input: Object.fromEntries(inputFilesPairs),
-  output: {
-    dir: outputDir,
-    sourcemap: true,
-    // add a hash on every file
-    entryFileNames: '[name]-[hash].js',
-    // sometimes, chunks are so small that they become counterproductive. Here we force some modules to be grouped in the same chunk.
-    manualChunks: manualChunkOptions,
-    // Rollup's import hoisting is not really useful here as we're applying a more aggressive import hoisting strategy with the deps-manifest.json
-    hoistTransitiveImports: false,
-  },
-  // fine-grained tree shaking options. Here we force tree shaking for leaflet and shoelace
-  treeshake: treeshakeOptions,
-  plugins: [
-    // output directory cleanup
-    clearPlugin({ outputDir }),
-    // transform .json files into ES6 modules. (Used by statuses library). TODO: we should get rid of this library
-    json(),
-    // add (and optimize) svg files used in all new URL('...', import.meta.url) pattern.
-    importMetaUrlAssetsPlugin(),
-    // convert CommonJS modules to ESM
-    commonjs(),
-    // support resolving module using Node resolution algorithm. TODO: figure out why we need that
-    resolve(),
-    // minify JS
-    terserPlugin(),
-    // minify HTML and CSS inside components
-    babelPlugin(),
-    // add and minify default theme
-    stylesAssetsPlugin({
-      transform: (stylesheet) => minifyStylesheet(stylesheet),
-    }),
-    // generate the deps-manifest.json file that will help our CDN to be smart
-    depsManifestPlugin({ packageVersion }),
-    // help us visualize and analyze the bundle size
-    visualizerPlugin({ outputDir, packageVersion }),
-  ],
-};
-
-function getVersion () {
-  const gitTag = process.env.GIT_TAG_NAME;
-  if (gitTag == null) {
-    throw new Error('Could not read version from git tag!');
+export function getCdnEntryName () {
+  const version = process.env.VERSION;
+  if (version == null) {
+    throw new Error('VERSION env var is required.');
   }
-  return gitTag.trim();
+
+  return toCdnEntryName(CDN_ENVIRONMENTS.release, version);
 }
```

Comment:
todo: This function is only used is this file, right? I would remove the `export` at least but I would go further by inlining these few lines before the default export, like this:

```js
const version = process.env.VERSION;
if (version == null) {
  throw new Error('VERSION env var is required.');
}

const cdnEntryName = toCdnEntryName(CDN_ENVIRONMENTS.release, version);

export default getCdnRollupConfig(cdnEntryName, true);
```